service/dap/server.go

```diff
@@ -822,7 +826,7 @@ func (s *Server) onLaunchRequest(request *dap.LaunchRequest) {
 		var cmd string
 		var out []byte
 		// Log debug binary build
-		if !((mode == "replay") && (s.config.Debugger.Backend == "core")) {
+		if !((mode == "replay") || (s.config.Debugger.Backend == "core")) {
```

Comment:
I still don't understand why you need to check if mode is "replay" or "core" if the if around all this code is only entered if mode is "debug" or "test".

Reply:
I got your original comment wrong on something that easy to see 🤦. Fixed!

Reply:
No worries. Odd blindspots happens to the best of us when we stare at the same code too much :)